source/scala.txt

```diff
@@ -17,10 +17,10 @@ MongoDB Scala Driver
 Introduction
 ------------
 
-This is the officially supported Scala driver for MongoDB.
-
-It's a modern idiomatic Scala driver with asynchronous and non-blocking IO.
-
+Welcome to the documentation site for the official MongoDB Scala 
+driver. You can add the driver to your application to work 
+asynchronously with MongoDB in Scala. Download it using ``sbt`` or 
+``maven``, or set up a runnable project by following our tutorials.
```

Comment:
For the other drivers like Swift, it seems like the pattern is to link to the tool and the official name, or the site from which they should download it. In this case, that means the change here is:
`sbt <https://www.scala-sbt.org/>`__
and
`Apache Maven <https://maven.apache.org/>`__

Reply:
Yes, good catch - updating